specs/discoverable_disk_image.md

```diff
@@ -18,3 +18,8 @@ The images use the GPT partition table verbatim, so it will not be redefined her
 a standard Linux filesystem (e.g.: squashfs), so again this will not be redefined here.
 The ([DPS](discoverable_partitions_specification.md)) defines the GUIDs to use and the format of the
 dm-verity signature partition's JSON content.
+
+## Image Version
+When versioning a DDI is appropriate, it is required to use the version format described in the
+[Version Format Specification](version_format_specification.md). As a separator between the version and
+other tokens in a string, the `_` character must be used.
```

Comment:
"""
If the DDI is versioned, the version format described in the [Version Format Specification](version_format_specification.md) must be used.
The underscore character (`_`) must  be used to separate the version from the name (?) of the image.
"""

Reply:
updated